templates/default/publish.js

```diff
@@ -300,7 +300,13 @@ function buildMemberNav(items, itemHeading, itemsSeen, linktoFn) {
                 itemsNav += '<li>' + linktoFn('', item.name) + '</li>';
             }
             else if ( !hasOwnProp.call(itemsSeen, item.longname) ) {
-                itemsNav += '<li>' + linktoFn(item.longname, item.name.replace(/^module:/, '')) + '</li>';
+                var displayName;
+                if (env.conf.templates.useLongnameInNav)
```

Comment:
Should be `env.conf.templates.default.useLongnameInNav`.

Also, please use braces in the if/else statement for consistency with the rest of the file.